README.md

```diff
@@ -79,6 +81,9 @@ graph.setData(nodes, links)
 | useQuadtree | Use the classic [quadtree algorithm](https://en.wikipedia.org/wiki/Barnes%E2%80%93Hut_simulation) for the Many-Body force. This property will be applied only on component initialization and it can't be changed using the `setConfig` method. <br /><br /> ⚠ The algorithm might not work on some GPUs (e.g. Nvidia) and on Windows (unless you disable ANGLE in the browser settings). | `false`
 | simulation | Simulation parameters and event listeners | See [Simulation configuration](#simulation_configuration) table for more details
 | events.onClick | Callback function that will be called on every canvas click. If clicked on a node, its data will be passed as a first argument, index as a second argument, position as a third argument and the corresponding mouse event as a forth argument: <code>(node: Node &vert; undefined, index: number &vert; undefined, nodePosition: [number, number] &vert; undefined, event: MouseEvent) => void</code> | `undefined`
+| events.onMouseMove | Callback function that will be called when mouse movement happens. If the mouse moves over a node, its data will be passed as a first argument, index as a second argument, position as a third argument and the corresponding mouse event as a forth argument: <code>(node: Node &vert; undefined, index: number &vert; undefined, nodePosition: [number, number] &vert; undefined, event: MouseEvent) => void</code> | `undefined`
+| events.onNodeMouseOver | Callback function that will be called when a node appears under the mouse because of mouse move event, zoom/drag event or simulation happens. The node data will be passed as a first argument, index as a second argument, position as a third argument and the corresponding mouse or zoom/drag event as a forth argument: <code>(node: Node, index: number, nodePosition: [number, number], event: MouseEvent &vert; D3ZoomEvent &vert; undefined) => void</code> | `undefined`
```

Comment:
because of mouse move event, zoom/drag event or simulation happens → as a result of a mouse event, zooming and panning, or movement of nodes.

zoom/drag event → zoom/pan event

The corresponding mouse or zoom/drag event → The corresponding mouse event or D3's zoom event